pkgs/applications/virtualization/kvmtool/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, fetchurl, lib }:
```

```suggestion
{ stdenv, fetchgit, lib }:
```
`fetchurl` is unused